sdk/test/metrics/BUILD

```diff
@@ -48,6 +48,22 @@ cc_test(
     ],
 )
 
+cc_test(
```

Comment:
`attributes_hashmap_test` and  `attributes_hashmap_benchmark` are missing in build.

Reply:
Thanks. Added.